src/quo/components/list_items/network_list/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns quo.components.list-items.network-list.view
+  (:require
+    [clojure.string :as string]
+    [quo.components.list-items.network-list.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [schema.core :as schema]))
+
+(defn- info
+  [{:keys [network-name label]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (quo.resources/get-network network-name)
+     :style  style/network-image}]
+   [rn/view
+    [text/text
+     {:weight          :semi-bold
+      :style           {:text-transform :capitalize}
+      :number-of-lines 1}
+     (if (string/blank? label) "-" label)]]])
+
+(defn- values
+  [{:keys [token-value fiat-value theme]}]
+  [rn/view {:style style/values-container}
+   [text/text
+    {:weight          :medium
+     :size            :paragraph-2
+     :number-of-lines 1}
+    token-value]
+   [text/text
+    {:style           (style/fiat-value theme)
+     :size            :paragraph-2
+     :number-of-lines 1}
+    fiat-value]])
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:network-name {:optional true} [:maybe keyword?]]
```

Comment:
If we know the possible network names, would be better to add the network names to the malli registry and use it here instead of `keyword?`. @J-Son89 what do you think?

Reply:
we no longer pass the network-name to the component based on [this conversation](https://github.com/status-im/status-mobile/pull/18547#discussion_r1456497723) above

Reply:
We have the image source schema already defined, so we can have `[:network-image :schema.common/image-source]` instead of `:int`. This way it can also cover the case when the source is passed as an uri. It's not necessary, but it's a nice-to-have.